### ISSUE TITLE
Ask "Are there any power outlets available for guests?" for tourism=alpine_hut as well. 

### ISSUE THREAD

**Issue by tiptoptom** [streetcomplete/StreetComplete#5943]
Alpine huts often don't have power for their guests. So it would be nice to ask this as well.

**Comment by westnordost** [comment]
@mnalis what do you think?

**Comment by matkoniecz** [comment]
> often don't have power for their guests

it may depend on country, in Poland basically all have

though their density is low and maybe tagging this obvious - for local tourists - info can be still useful?

**Comment by kmpoppe** [comment]
I'd say makes sense to me. There aren't any alpine huts in my area (Bremen, Germany is the Plains Biome equivalent of Minecraft) but having been on the odd hiking trip in Scotland where trails may last days, I think this information is useful for tourists.

**Comment by mnalis** [comment]
> @mnalis what do you think?

**TL;DR:** Dunno. Only has very tiny usage currently, but could be useful.

---

I don't have much experience with alpine huts :man_shrugging: 

However, [Overpass](https://overpass-turbo.eu/s/1S9c) says that there currently are **only 21** `power_supply=*` usages worldwide (which is only `0.15%` of [tourism=alpine_hut](https://taginfo.openstreetmap.org/tags/tourism=alpine_hut#overview)), so it would be SC pushing this tag combination if implemented. Other than that, it does not feel spammy (not that many alpine huts) and could be useful.

Wiki for [tourism=alpine_hut](https://wiki.openstreetmap.org/wiki/Tag%3Atourism%3Dalpine_hut) also doesn't mention `power_supply=*` (which is used by the mentioned  _"Are there any power outlets available for guests?"_ quest). It does however mention [electricity=*](https://wiki.openstreetmap.org/wiki/Key:electricity) to define where does electricity come from (`grid`/`generator`/`solar`/`no`/...) if it is available. 

One should note that `electricity` is somewhat different tag from `power_supply` - it also covers things like available lights, electrical heating, hot water for shower, air conditioning etc, and yes, quite often (but not necessarily always) also general sockets (i.e. `power_supply`). `electricity`, while documented, is only slightly more used, **54** cases worldwide (out of 13k+ of alpine huts, so `0.39%`)

**Comment by westnordost** [comment]
Hmm... interesting.

But it is true that alpine huts are sometimes somewhat in the wilderness, meaning that it is not a given that there will be (steady) supply of electricity. The mention of `electricity=*` on that wiki page is a good indicator for that. So, I would say, it can be included.

Although, of course, the hint may be a bit confusing, as it refers to the camping ground. But, I think this is a minor thing.
